{"1":{"str":"php_pinpoint"},"2":{"str":"PHPApplication"},"3":{"i64":1539930010095},"4":{"str":"ABhwaHBfcGlucG9pbnTv4+zX6Cwe"},"7":{"i64":1755065569094089873},"8":{"i64":-1},"9":{"i64":1539930030418},"10":{"i32":1},"11":{"str":"/test_recursive_exception1.php"},"12":{"i16":1500},"13":{"str":"127.0.0.1:8989"},"14":{"str":"127.0.0.1"},"15":{"lst":["rec",2,{"1":{"i32":999},"2":{"rec":{"1":{"str":"/home/liumingyi/git/pinpoint-c-agent/simulate/testcase/php/trace/test_recursive_exception1"}}}},{"1":{"i32":300},"2":{"rec":{"11":{"rec":{"1":{"i64":1539930030418},"2":{"i32":2},"3":{"i32":0}}}}}}]},"16":{"i16":0},"17":{"i32":1},"18":{"lst":["rec",3,{"7":{"i64":1755065569094089873},"8":{"i16":2},"9":{"i32":1},"10":{"i32":1},"12":{"i16":1501},"14":{"lst":["rec",1,{"1":{"i32":-1},"2":{"rec":{"1":{"str":"Array\n(\n    [0] =&gt; 0\n)\n"}}}}]},"15":{"i32":3},"16":{"i64":-1},"25":{"i32":-23},"26":{"rec":{"1":{"i32":-1},"2":{"str":"Fatal error: n < 1 in /home/liumingyi/git/pinpoint-c-agent/simulate/web/php/test_recursive_exception1.php on line 20"}}}},{"7":{"i64":1755065569094089873},"8":{"i16":1},"9":{"i32":1},"10":{"i32":1},"12":{"i16":1501},"14":{"lst":["rec",1,{"1":{"i32":-1},"2":{"rec":{"1":{"str":"Array\n(\n    [0] =&gt; 1\n)\n"}}}}]},"15":{"i32":2},"16":{"i64":-1},"25":{"i32":-23}},{"7":{"i64":1755065569094089873},"8":{"i16":0},"9":{"i32":1},"10":{"i32":1},"12":{"i16":1501},"14":{"lst":["rec",1,{"1":{"i32":-1},"2":{"rec":{"1":{"str":"Array\n(\n    [0] =&gt; 2\n)\n"}}}}]},"15":{"i32":1},"16":{"i64":-1},"25":{"i32":-23}}]},"25":{"i32":-13},"26":{"rec":{"1":{"i32":-2},"2":{"str":"Fatal error: Uncaught exception 'Exception' with message 'n < 1' in /home/liumingyi/git/pinpoint-c-agent/simulate/web/php/test_recursive_exception1.php:20\nStack trace:\n#0 /home/liumingyi/git/pinpoint-c-agent/simulate/web/php/test_recursive_exception1.php(21): test_cumsum_e1(0)\n#1 /home/liumingyi/git/pinpoint-c-agent/simulate/web/php/test_recursive_exception1.php(21): test_cumsum_e1(1)\n#2 /home/liumingyi/git/pinpoint-c-agent/simulate/web/php/test_recursive_exception1.php(24): test_cumsum_e1(2)\n#3 {main}\n  thrown in /home/liumingyi/git/pinpoint-c-agent/simulate/web/php/test_recursive_exception1.php on line 20"}}}}